src/resolver.ts

```diff
@@ -565,7 +565,7 @@ export class Resolver extends DiagnosticEmitter {
     if (!(typeArgumentNodes && typeArgumentNodes.length == 1)) {
       if (reportMode == ReportMode.REPORT) {
         this.error(
-          DiagnosticCode.Expected_0_type_arguments_but_got_1,
+          DiagnosticCode.Type_0_has_no_call_signatures,
```

Comment:
Previous error message was correct in that it reports missing type arguments on the builtin itself.

Reply:
Got it. Switched back.